server/src/test/java/io/crate/integrationtests/CreateTableIntegrationTest.java

```diff
@@ -145,5 +147,50 @@ public void test_enforce_soft_deletes_on_partitioned_tables() {
 
     }
 
+    @Test
+    public void test_column_positions_from_create_table() {
```

Comment:
If I'm not mistaken we already have this test case.  Could you double check?

Reply:
There are similar ones in `DDLIntegrationTest` but I was thinking that the create table stmt from https://github.com/crate/crate/issues/12630#issue-1276791467 could turn into a test as well as to cover a little more. (named index was missing the position as well)